src/packages/solid/components/ListItem/ListItem.styles.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2024 Comcast Cable Communications Management, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { NodeStyles } from '@lightningjs/solid';
+import theme from 'theme';
+import type { Tone } from '../../types/types.js';
+import type { ComponentStyleConfig, NodeStyleSet, TextStyleSet } from '../../types/types.js';
+import { makeComponentStyles } from '../../utils/index.js';
+import { getWidthByUpCount } from '../../utils/getWidthByUpcount.js';
+
+export interface ListItemStyles {
+  tone: Tone,
+  Container: NodeStyleSet;
+  Title: TextStyleSet;
+  Description: TextStyleSet;
```

Comment:
any property we're returning to the component that isn't included inside of `NodeStyles` needs to be added to NodeStyleSet
ex.
```
Container: NodeStyleSet<{ paddingX: number }>;
```